ftp.go

```diff
@@ -28,6 +28,15 @@ const (
 	EntryTypeLink
 )
 
+// TransferType denotes the formats for transfering Entries.
+type TransferType string
+
+// The different transfer types
+const (
+	TransferTypeImage = "I"
```

Comment:
Can we rename `TransferTypeImage` to `TransferTypeBinary` ?

Reply:
Of course -and done. :-)